appengine/firebase-tictactoe/README.md

```diff
@@ -0,0 +1,54 @@
+# Tic Tac Toe on Google App Engine Standard using Firebase
+
+This directory contains a project that implements a realtime two-player game of
+Tic Tac Toe on [Google App Engine] Standard, using the [Firebase] database
+for realtime notifications when the board changes.
+
+[Firebase]: https://firebase.google.com
+[Google App Engine]: https://cloud.google.com/appengine
+
+## Prerequisites
+
+* Install [Apache Maven][maven]
+* Create a project in the [Firebase Console][fb-console]
+* Install the [Google Cloud SDK][sdk]
+* Download [service account credentials][creds] and move it to
+  `src/main/webapp/resources/credentials.json`.
+* In the [Overview section][fb-overview] of the Firebase console, click 'Add
+  Firebase to your web app' and replace the contents of the file
+  `src/main/webapp/resources/firebase_config.html` with that code snippet.
+
+[fb-console]: https://console.firebase.google.com
+[sdk]: https://cloud.google.com/sdk
+[creds]: https://console.firebase.google.com/iam-admin/serviceaccounts/project?project=_&consoleReturnUrl=https:%2F%2Fconsole.firebase.google.com%2Fproject%2F_%2Fsettings%2Fgeneral%2F
+[fb-overview]: https://console.firebase.google.com/project/_/overview
+
+
+## Run the sample
+
+* To run the app locally:
+
+    ```sh
+    $ mvn appengine:run
+    ```
+
+## Troubleshooting
+
+* If you see the error `Google Cloud SDK path was not provided ...`:
+    * Make sure you've installed the [Google cloud SDK][sdk]
+    * You may have installed it in a non-standard path. In that case, set the
```

Comment:
Good point - you might mention that the Google Cloud SDK needs to be on your `PATH`, if not, you can fix it by adding...